src/dune/coq_rules.ml

```diff
@@ -409,16 +409,20 @@ let install_rules ~sctx ~dir s =
     in
     Dir_contents.coq dir_contents
     |> Coq_sources.library ~name
-    |> List.map ~f:(fun (vfile : Coq_module.t) ->
+    |> List.concat_map ~f:(fun (vfile : Coq_module.t) ->
+           let to_path f = Path.reach ~from:(Path.build dir) (Path.build f) in
```

Comment:
It seems to me that this function could be useful also for OCaml once it gets `qualified` subdir support?

Should we move it to `Path` @rgrinberg ?

Reply:
For now, let's keep it here. In fact, we are trying to avoid adding dune specific stuff to Path and make it a more general purpose module.

Reply:
Ok, I was thinking that'd be a general function tho, "relocate" ? My main gripe is that the current implementation doesn't seem very efficient, but not a big deal indeed.